packages/ui/src/components/va-pagination/VaPagination.vue

```diff
@@ -280,6 +280,7 @@ export default defineComponent({
       preset: props.buttonsPreset,
       color: props.color,
       borderColor: props.borderColor,
+      round: props.rounded,
```

Comment:
Rounded state is controlled by VaPagination, no?

Reply:
No, it don't.

Reply:
But this state is set by VaPagination classes and that was also for purpose :)

<img width="494" alt="image" src="https://user-images.githubusercontent.com/64714442/203799848-0eba5b3e-c2e4-4b19-9b60-1d5f8e85c0ca.png">